sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -1944,6 +1996,57 @@ def sdc_pandas_dataframe_iat_impl(self):
     return sdc_pandas_dataframe_iat_impl
 
 
+@sdc_overload_attribute(DataFrameType, 'loc')
+def sdc_pandas_dataframe_loc(self):
+    """
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+
+    Pandas API: pandas.DataFrame.loc
+
+    Limitations
+    -----------
+    - Parameter ``'name'`` in new DataFrame can be String only
```

Comment:
What does the parameter `name` mean?

Reply:
In this case it means that result series (if it series) has name string
Maybe need change limitation to more understanding

Reply:
What is difference between Pandas and SDC in case of `name` of the Series?

Reply:
Doesn't we support numeric name for the Series?

Reply:
Yes, we support only string name series
And if base series containe numeric name, we change it into string